pkg/framework/provider/crypto/crypto_test.go

```diff
@@ -0,0 +1,150 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package crypto
+
+import (
+	"testing"
+
+	"github.com/google/tink/go/aead"
+	"github.com/google/tink/go/keyset"
+	"github.com/google/tink/go/signature"
+	aeadsubtle "github.com/google/tink/go/subtle/aead"
+	"github.com/stretchr/testify/require"
+	chacha "golang.org/x/crypto/chacha20poly1305"
+
+	cryptoapi "github.com/hyperledger/aries-framework-go/pkg/framework/spi/crypto"
+)
+
+func TestCrypto(t *testing.T) {
+	// Assert that TinkCrypto implements the TinkCrypto interface.
+	var _ cryptoapi.Crypto = (*TinkCrypto)(nil)
+}
```

Comment:
same, it is compiler check not a test case

Reply:
removed test function